.pre-commit-config.yaml

```diff
@@ -28,15 +27,16 @@ repos:
     hooks:
       - id: flake8
         args: ["--max-line-length=120"]
-        exclude: ^.*__init__\.py$|^tests/
+        exclude: up42/__init__.py|tests/conftest.py # ignore pytest fixtures
```

Comment:
NIT: for __init__.py it's recommended to use the imported modules in __all__

Reply:
I did not get it.
Can you elaborate?

Reply:
e.g. `__all__ =[obj.__name__ for obj in [Asset, Auth, Catalog, ...]]` in `__init__.py`. That would make the export of the objects to top level namespace more explicit and there will be no complaints related to unused imported names

Reply:
nice!
didn't knew about this. Thanks